src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,80 @@
+package main.java.com.google.sps.servlets;
+
+import com.google.api.gax.paging.Page;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+
+import java.util.*;
+
+@WebServlet("/comment")
+@MultipartConfig
+public class CommentsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    response.setContentType("text/html;");
+    response.getWriter().println("<h1>Show Comments.</h1>");
+    response.getWriter().println("<ul>"); // <ul> --> 'unordered list element'
+    
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+
+    // Setting an order by time posted
+    Query<Entity> query = Query.newEntityQueryBuilder()
+      .setKind("Comment")
+      .setOrderBy(OrderBy.desc("timestamp"))
+      .build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    // Basicallly stored like(?) a list. This is printing comments.
```

Comment:
*Easy to follow code
 *You might want to consider writing the comment to be a little bit more clear. For instance,  the question mark symbol